modDbImpl/src/org/aion/db/impl/DatabaseFactory.java

```diff
@@ -69,6 +71,8 @@
     public static final String PROP_BLOCK_SIZE = "block_size";
 
     public static final String PROP_WRITE_BUFFER_SIZE = "write_buffer_size";
+    public static final String PROP_READ_BUFFER_SIZE = "read_buffer_size";
```

Comment:
The property `"read_buffer_size"` should be added to `org.aion.mcf.config.CfgDb.java` as well.